projects/ion/src/lib/accordion/accordion.component.spec.ts

```diff
@@ -9,7 +9,7 @@ import { IonAccordionComponent } from './accordion.component';
 
 @Component({
   template: `<ion-accordion [name]="name">
-    <p data-testID="ion-accordion__main-paragraph">Context Main</p>
+    <p data-testId="ion-accordion__main-paragraph">Context Main</p>
```

Comment:
```suggestion
    <p data-testid="ion-accordion__main-paragraph">Context Main</p>
```

https://testing-library.com/docs/queries/bytestid/